configure

```diff
@@ -74,6 +74,7 @@ Usage: $0 [OPTION]... [VAR=VALUE]...
                            (Zeek uses an embedded version by default)
     --with-caf=PATH        path to C++ Actor Framework install root
                            (a Broker dependency that is embedded by default)
+    --with-libkqueue=PATH  path to libkqueue install root
```

Comment:
Let's add: ``(Zeek uses an embedded version by default)`, similar to other deps that we ship